stylesheets/_layout.settings.scss

```diff
@@ -352,16 +358,31 @@
  * Content area
  */
 
+.ie9 .tab__container.has-settings {
+    overflow-x: auto;
+}
+
 .tab__container.has-settings .tab__content {
     @include respond-min($screen-medium) {
         margin-bottom: $line-height-small;
-        padding-left: $padding-small-horizontal * 2;
+        -webkit-flex: 1 1 auto;
+        -moz-flex: 1 1 auto;
+        -ms-flex: 1 1 auto;
+        -o-flex: 1 1 auto;
+        flex: 1 1 auto;
```

Comment:
❓ What are these for?

Reply:
The prefixes or the longhand?

Reply:
The usage of `flex` 😄

Reply:
The autoprefixer should take care of the vendor stuff shouldn't it?

Reply:
Found this, it's a weird bug in IE10 which caused the table to extend even further at wide viewport widths. Moving flex to the container doesn't resolve the issue, but I've removed all but `-ms-flex` to just target IE with this.

**Before**
![ie10-before](https://user-images.githubusercontent.com/18653/48624820-5d5c3600-e9a5-11e8-84d1-ae0ee5691fe7.gif)

**After**
![ie10-after](https://user-images.githubusercontent.com/18653/48624833-63521700-e9a5-11e8-983d-c0ee8da90d9d.gif)

Reply:
Very weird